utils/vif.c

```diff
@@ -769,6 +808,11 @@ parse_long_opts(int option_index, char *opt_arg)
         vr_op = SANDESH_OP_DUMP;
         break;
 
+    case STATS_OPT_INDEX:
+        core = (short)strtoul(opt_arg, NULL, 0);
+        vr_op = SANDESH_OP_DUMP;
```

Comment:
Can this be a OP_GET depending on whether user wants all interface stats or just one interface?